docs/sdk/teamsfx.querywithtoken.md

```diff
@@ -0,0 +1,43 @@
+<!-- Do not edit this file. It is automatically generated by API Documenter. -->
+
+[Home](./index.md) &gt; [@microsoft/teamsfx](./teamsfx.md) &gt; [queryWithToken](./teamsfx.querywithtoken.md)
+
+## queryWithToken() function
+
+Users execute query with SSO or Access Token.
+
+<b>Signature:</b>
+
+```typescript
+export declare function queryWithToken(context: TurnContext, config: AuthenticationConfiguration | null, scopes: string | string[], logic: (token: TeamsMsgExtTokenResponse) => Promise<any>): Promise<MessagingExtensionResponse | void>;
+```
+
+## Parameters
+
+|  Parameter | Type | Description |
+|  --- | --- | --- |
+|  context | TurnContext | The context object for the current turn. |
+|  config | [AuthenticationConfiguration](./teamsfx.authenticationconfiguration.md) \| null | User custom the message extension authentication configuration. |
+|  scopes | string \| string\[\] | The list of scopes for which the token will have access. |
+|  logic | (token: [TeamsMsgExtTokenResponse](./teamsfx.teamsmsgexttokenresponse.md)<!-- -->) =&gt; Promise&lt;any&gt; | The user execution code with SSO or Access token. |
+
```

Comment:
Maybe the explanation can be changed to: `Business logic when executing the query in message extension with SSO or access token.`

Reply:
thanks, updated.